src/hazelcore/UHTyp.re

```diff
@@ -5,6 +5,7 @@ type operator = Operators_Typ.t;
 type t = opseq
 and opseq = OpSeq.t(operand, operator)
 and operand =
+  | TyVar(TyVarErrStatus.t, TyVar.t)
```

Comment:
Add InvalidText as a top level entry here to follow UHExp instead of putting it all in TyVar

Reply:
fixed by 82b935785 and f344ae350 and 22e9d99e0